src/modlunky2/mem/memrauder/model.py

```diff
@@ -378,6 +388,16 @@ def from_bytes(self, buf: bytes, mem_ctx: MemContext) -> T:
         except Exception as err:
             raise ScalarCValueConstructionError(self.path, mem_value) from err
 
+    def to_bytes(self, py_val: T) -> bytes:
+        c_val = None
+        try:
+            c_val = self.c_type(py_val)
```

Comment:
Python doesn't have block scoping so there's no need to initialize `c_val` to `None`. `None` is never valid for this type anyways as far as I can tell

Reply:
Good point.